src/foam/u2/view/UnstyledTableView.js

```diff
@@ -806,6 +801,27 @@ foam.CLASS({
   ]
 });
 
+foam.CLASS({
+  package: 'foam.u2.view',
+  name: 'TableViewActionRefinement',
+  refines: 'foam.core.Action',
+  properties: [
+    {
+      class: 'Boolean',
+      name: 'columnHidden'
```

Comment:
@kgrgreer Since those are actions so it should be shown in invoice but not selectable in column. If I set availablePermissions then user may not able to send a draft payment because this action need permission. So this is same as the property which we do want use them but we just don't want display them in the tableView

Reply:
@kgrgreer And the Action isn't excluded from the column, but that can be a solution as well. Are you prefer just remove action from column?

Reply:
@olhabn is there a reason why Actions was added  in column?